server/api/index.js

```diff
@@ -3,8 +3,14 @@ const router = express.Router();
 
 const users = require("./users");
 const status = require("./status");
+const cities = require("./cities");
+const skills = require("./skills");
+const newOpportunity = require("./newOpportunity");
 
 router.use("/users", users);
 router.use("/status", status);
+router.use("/cities", cities);
+router.use("/skills", skills);
+router.use("/newOpportunity", newOpportunity);
```

Comment:
rename them to `/opportunities` and make sure the client is also updated to `POST /opportunities` .. also update the file name to be `opportunities.js`